<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogTimeFollower</class>
 <widget class="QDialog" name="DialogTimeFollower">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>144</width>
    <height>99</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1</width>
    <height>1</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="toolTip">
   <string>You can move me!</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="horizontalSpacing">
    <number>2</number>
   </property>
   <item row="0" column="0" rowspan="2" colspan="2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>1</number>
     </property>
     <item>
      <widget class="MoveableLabel" name="label_2">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../EasyTimeTracker.qrc">:/images/tt_title_bar.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="MoveableLabel" name="labelDayRemaining">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>35</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>24</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>You can move me!</string>
         </property>
         <property name="text">
          <string>00:00</string>
         </property>
         <property name="textFormat">
          <enum>Qt::RichText</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="MoveableLabel" name="label">
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="toolTip">
          <string>You can move this dialog clicking on me!</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../EasyTimeTracker.qrc">:/images/tt_2black_coffees_bk_gnd_30x30.png</pixmap>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="pushButtonStart">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <selectedon>:/images/tt_start.png</selectedon>
          </iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonStop">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <selectedon>:/images/tt_stop.png</selectedon>
          </iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonPause">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <selectedon>:/images/tt_pause.png</selectedon>
          </iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>10</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonHideWindow">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <selectedon>:/images/tt_cross.png</selectedon>
          </iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>3</number>
       </property>
       <item>
        <widget class="MoveableLabel" name="labelMonthWorked">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>Mth: 000:00</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignRight">
        <widget class="MoveableLabel" name="labelBalance">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>Bal: 000:00</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSlider">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>10</height>
        </size>
       </property>
       <property name="value">
        <number>99</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::NoTicks</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MoveableLabel</class>
   <extends>QLabel</extends>
   <header>DialogTimeFollower/dialogtimefollower.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../EasyTimeTracker.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButtonHideWindow</sender>
   <signal>clicked()</signal>
   <receiver>DialogTimeFollower</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>203</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>165</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
